packages/medusa/src/migrations/1649775522087-add_batch_job_model.ts

```diff
@@ -5,38 +5,15 @@ export class addBatchJobModel1649775522087 implements MigrationInterface {
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(
-        `CREATE TABLE "batch_job"
```

Comment:
Suggestion: since this is an existing migration that has been updated, i would suggest to undo those changes and to look at the next suggestion if it can solve the problem